executor/sort.go

```diff
@@ -247,7 +247,8 @@ func (e *SortExec) fetchRowChunks(ctx context.Context) error {
 				e.rowChunks.GetMemTracker().SetLabel(rowChunksLabel)
 				e.rowChunks.GetDiskTracker().AttachTo(e.diskTracker)
 				e.rowChunks.GetDiskTracker().SetLabel(rowChunksLabel)
-				e.spillAction.ResetRowContainer(e.rowChunks)
+				e.spillAction = e.rowChunks.ActionSpill()
```

Comment:
Should `ActionSpillForTest` be involved here? Should a unit test be covered with this situation?